package.json

```diff
@@ -44,7 +42,9 @@
     "babel-core": "^7.0.0-bridge.0"
   },
   "devDependencies": {
-    "@babel/runtime": "^7.7.4"
+    "@apollo/react-testing": "^3.1.3",
+    "@babel/runtime": "^7.7.4",
+    "react-intl": "^3.12.0"
```

Comment:
Why react-intl and apollo are here?

Reply:
oh! sorry, I forgot to remove them. I added them to test this locally, because yarn was complaining about peer dependencies.